packages/darklang/stdlib/int8.dark

```diff
@@ -140,6 +140,6 @@ module Darklang =
       let toString (int8: Int8) : String = Builtin.Int8.toString int8
 
 
-      /// Converts an <type Int8> to a <type Int64>. Returns {{None}} if the value is less than -128 or greater than 127.
+      /// Converts a <type Int8> to a <type Int64>. Returns {{None}} if the value is less than -128 or greater than 127.
```

Comment:
I think it was better before?
When presented in the editor/wherever, this should read as "Converts an Int8 to an Int64. Returns ...", with `Int8` and `Int64` highlighted in a special way.

Reply:
True, I'll fix it